elixir/apps/web/lib/web/live/resources/index.ex

```diff
@@ -58,8 +65,16 @@ defmodule Web.Resources.Index do
         Resources
       </:title>
       <:help>
-        Resources define the subnets, hosts, and applications for which you want to manage access. You can manage Resources per Site
-        in the <.link navigate={~p"/#{@account}/sites"} class={link_style()}>Sites</.link> section.
+        <p class="mb-2">
+          Resources define the subnets, hosts, and applications for which you want to manage access. You can manage Resources per Site
+          in the <.link navigate={~p"/#{@account}/sites"} class={link_style()}>Sites</.link> section.
+        </p>
+        <p :if={Domain.Accounts.internet_resource_enabled?(@account) && @internet_site}>
+          The Internet Resource can now be managed in the
+          <.link navigate={~p"/#{@account}/sites/#{@internet_site}"} class={link_style()}>
+            Internet Site.
+          </.link>
```

Comment:
Just curious if this will be a temporary message?

Reply:
Yeah

Reply:
Will do another pass post-march-15